contributors/py-helix-admin/helix/statemodeldefs.py

```diff
@@ -0,0 +1,39 @@
+from ordereddict import OrderedDict
+
+# These essentially come from the java classes defined here. It is cheesey and should probably come from a configuration file.
+# https://github.com/linkedin/helix/blob/master/helix-core/src/main/java/com/linkedin/helix/tools/StateModelConfigGenerator.java
+
```

Comment:
I use ordereddict because I think there are places where ordering matters with regards to the data in zookeeper. Using ordereddict ensures compatibility as python dictionaries (and the json parser) do not guarantee ordering. Let me know if the ordering requirement can be dropped and I can easily rewrite this.